codebase/config/sync/views.view.draft_content.yml

```diff
@@ -0,0 +1,815 @@
+uuid: ad739a60-6490-4cda-973f-ddaf3d18020d
```

Comment:
Does this mean the other view for Manage Studies can be deleted?

Reply:
I have kept it undeleted but in the disabled views section.